Dockerfile

```diff
@@ -306,10 +302,10 @@ RUN chmod +x /usr/local/bin/doctor && \
     chmod +x /usr/local/bin/patch-create-missing-schedules && \
     chmod +x /usr/local/bin/maintenance && \
     chmod +x /usr/local/bin/volume-sync && \
-    chmod +x /usr/local/bin/usage && \
     chmod +x /usr/local/bin/install && \
     chmod +x /usr/local/bin/migrate && \
     chmod +x /usr/local/bin/realtime && \
+    chmod +x /usr/local/bin/executor && \
```

Comment:
We don't need this anymore . The executor has been moved to open runtimes

Reply:
removed